spec/features/user_changes_password_spec.rb

```diff
@@ -0,0 +1,32 @@
+feature "User changes password" do
+  before :each do
```

Comment:
Instead, I prefer to see each test set up its own data. See https://github.com/codecation/trailmix/blob/master/spec/features/user_changes_settings_spec.rb#L2 for example.